roles/openshift_health_checker/test/mixins_test.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class NotContainerizedCheck(NotContainerizedMixin, OpenShiftCheck):
+    name = "not_containerized"
+    run = NotImplemented
+
+
+@pytest.mark.parametrize('task_vars,expected', [
+    (dict(openshift=dict(common=dict(is_containerized=False))), True),
+    (dict(openshift=dict(common=dict(is_containerized=True))), False),
+])
+def test_is_active(task_vars, expected):
+    assert NotContainerizedCheck.is_active(task_vars) == expected
+
+
+def test_is_active_missing_config():
```

Comment:
> e.g. "is_containerized requests an undefined task_var, resulting in an exception that should be caught and re-raised as OpenShiftCheckException"

This is an implementation detail. Too much specificity makes the tests brittle.

The intention here is more on what the test name says: if we call `is_active` without proving `task_vars`, we expect an `OpenShiftCheckException` that talks about `'is_containerized'`.

I don't know what I had in mind when I wrote "missing config"... perhaps `...missing_task_vars` is better?